{"uuid": "2cf55e57-6f8c-4c45-9620-269afdbffaec", "children": ["7919605c-36f0-4181-81f1-7aea65c38202", "561c432e-8165-48e7-812a-afd77498893b", "7815c048-4b63-4c6c-9c0a-4b929ce26ab8", "72a7ccf7-693d-41a5-b1f8-1b74b7d3c644", "4a3acfca-6d32-4c6b-bff4-abd772befa68", "738ac114-4a86-4013-aa90-bd1455cdb8f9", "19270bc4-232b-43f0-9c3b-5bc480bdfdd9", "a37dc241-da74-4adc-8751-6799fd2c1a29", "4fe971e3-057e-4c73-95a8-230471d49244", "33a1f5d1-4f4e-4f2d-8fb5-d8cd538884ed", "28ec1c7c-8f12-46c9-937e-3cfd7e10dd10", "b60f509f-ffb8-45ed-840b-96c7ffabcc4b", "a9c44fc1-2ac2-452d-b525-34dc8ddd540d", "93cd90c8-c90b-44ae-8580-bebe8884912f", "a0233c32-8960-4ae2-941d-a678053defc5", "2d5a5732-2d97-4210-82ce-8dbff13b90f7", "4926ae71-a87d-4bcf-8834-b95989bdf3eb", "6b93172a-2600-41c0-808d-265c32ca83c9", "93cb5b62-6d60-4acf-a313-7c07f5a93d49", "ce854f68-74b0-4ec2-8d5c-7f5ae5b3b3a0", "61675561-4368-4daf-8b92-d6a07c666349", "a736c371-8a34-4576-a4ad-c023da29fc2c", "58470c66-39a0-4fbd-ad7e-58ef6ce971ae", "b06ad2e0-e14e-4c4c-8155-3fb8b4089d92", "727934f2-d9da-4d9b-ae36-578c8eb7ee4a", "48be9bc6-60fc-4bf1-867d-6a490617d930"], "befores": [{"name": "setup_teardown", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=134.0.6998.90)\nStacktrace:\n\tGetHandleVerifier [0x00007FF7D324FE65+26629]\n\t(No symbol) [0x00007FF7D31B6030]\n\t(No symbol) [0x00007FF7D304931A]\n\t(No symbol) [0x00007FF7D3045D10]\n\t(No symbol) [0x00007FF7D30368C9]\n\t(No symbol) [0x00007FF7D3038638]\n\t(No symbol) [0x00007FF7D3036BD6]\n\t(No symbol) [0x00007FF7D3036656]\n\t(No symbol) [0x00007FF7D303631A]\n\t(No symbol) [0x00007FF7D3033E9F]\n\t(No symbol) [0x00007FF7D303478C]\n\t(No symbol) [0x00007FF7D304D16A]\n\t(No symbol) [0x00007FF7D30F102E]\n\t(No symbol) [0x00007FF7D30C7AAA]\n\t(No symbol) [0x00007FF7D30F0169]\n\t(No symbol) [0x00007FF7D30C7883]\n\t(No symbol) [0x00007FF7D3090550]\n\t(No symbol) [0x00007FF7D3091803]\n\tGetHandleVerifier [0x00007FF7D35A72DD+3529853]\n\tGetHandleVerifier [0x00007FF7D35BDA42+3621858]\n\tGetHandleVerifier [0x00007FF7D35B24F3+3575443]\n\tGetHandleVerifier [0x00007FF7D331B79A+860474]\n\t(No symbol) [0x00007FF7D31C08AF]\n\t(No symbol) [0x00007FF7D31BCBE4]\n\t(No symbol) [0x00007FF7D31BCD86]\n\t(No symbol) [0x00007FF7D31AC2E9]\n\tBaseThreadInitThunk [0x00007FFEE8F2E8D7+23]\n\tRtlUserThreadStart [0x00007FFEE957BF6C+44]\n\n", "trace": "  File \"C:\\Users\\GRL\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\GRL\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\GRL\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\GRL\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\GRL\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\GRL\\PycharmProjects\\25W automation\\Project-24\\tests\\test_config_panelchecks.py\", line 145, in setup_teardown\n    driver = browser.way1(URL)\n  File \"C:\\Users\\GRL\\PycharmProjects\\25W automation\\Project-24\\utilities\\browser_setup.py\", line 37, in way1\n    self.driver.get(url)\n    ~~~~~~~~~~~~~~~^^^^^\n  File \"C:\\Users\\GRL\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 393, in get\n    self.execute(Command.GET, {\"url\": url})\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\GRL\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\GRL\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1742478973880, "stop": 1742478980374}], "start": 1742478973880, "stop": 1742478983613}